File: julia/9.jl
ans = 0
for a = 1:1000
	for b = 1:1000
		for c = 1:1000
			if a^2 + b^2 === c^2 && a + b + c === 1000
				global ans = a*b*c
				break
			end
		end
	end
end

println(ans)
# 31875000

File: solutions/9.c
#include <stdio.h>

int main() {
  int ans = 0;

  for (int a = 1; a <= 1000; a++) {
    for (int b = 1; b <= 1000; b++) {
      for (int c = 1; c <= 1000; c++) {
        if (a * a + b * b == c * c && a + b + c == 1000) {
          ans = a * b * c;
          break;
        }
      }
    }
  }

  printf("%d\n", ans);
  return 0;
}
// 31875000%
